### ISSUE TITLE
Mac Intel 10.13.6 | Blender 2.93

### ISSUE THREAD

**Issue by circuitbone** [Gorgious56/MazeGenerator#5]
After first press to generate a maze, I am presented with this message.

![Screen Shot 2021-11-24 at 1 53 59 pm](https://user-images.githubusercontent.com/47682220/143163391-79bf20d7-6351-4e43-b206-01f74eb0f173.png)

Once I have generated a maze, I cannot trash or create a new one.

**Comment by circuitbone** [comment]
Ok after some more tinkering I got a stable session.

**Comment by Gorgious56** [comment]
Yeah unfortunately this add-on relies quite heavily on custom drivers and modifiers so you have to allow execution of scripts. You can allow only for this session if you don't check "Permanently allow execution of scripts" but you'll have to do it again every time your relaunch blender.

That's limitation of how it was designed.

I can assure you this add-on does not execute malicious code on your computer, the worse that can happen is your Blender instance crashes or hangs because of a bug or if you set up too high values in the maze dimensions.

If you do allow permanent execution of scripts, you can always go back in the preferences and you can even setup certain directories to not allow execution of scripts, for example in your /downloads folder for files you download from unknow sources.

**Comment by circuitbone** [comment]
[1] Thanks for the input mate, I was documenting my experiences in case it helps you make a better plugin.

[2] Can you elaborate more on what it is enabling?
 


[3] Do you have a form for feature requests?

- Starting point object instantiated and placed at the start.
- End point object instantiated and placed at the end.
- Larger Rooms with Arches or Doors at the given entrances and exits.

**Comment by circuitbone** [comment]
https://imgur.com/gallery/U5yoODH/ I get a bad link address on this url

I assume it is a copy of the content on Art Station.

**Comment by circuitbone** [comment]
Technique : 

Is there a tutorial or explanation on how I can apply a maze to a given mesh?
How is the height managed in this demonstration?

https://user-images.githubusercontent.com/25156105/80923347-6ef4f180-8d83-11ea-9442-392e6ca7927a.JPG

**Comment by circuitbone** [comment]
Did you apply a shrinkwrap to get the above effect?
